examples/conduction-velocity-benchmark.jl

```diff
@@ -75,6 +75,9 @@ timestepper = OS.LieTrotterGodunov((
         reaction_threshold=0.1f0,
     ),
 ))
+controller = Thunderbolt.ReactionTangentController(0.5, 1.0, (0.01, 0.3), 0.0, 0.0)
+
+timestepper = Thunderbolt.AdaptiveOperatorSplittingAlgorithm(_timestepper, controller)
```

Comment:
The style of the tutorials condenses these with indentations, i.e.
```suggestion
timestepper = Thunderbolt.AdaptiveOperatorSplittingAlgorithm(
    OS.LieTrotterGodunov(
        ...
     ),
     ReactionTangentController(
         ...
     )
```